apps/music/test/marionette/Player_test.js

```diff
@@ -185,4 +188,74 @@ marionette('Music player tests', function() {
     });
   });
 
+  suite('Player navigation', function() {
+    test('Check that the back button works', function() {
+      // the navigation test back from the tab
+      function tabNavTest() {
+        assert.equal(client.findElement(Music.Selector.viewsList).
```

Comment:
nit: the line break after the . makes this code hard to read. If you need to break because of line length, the second line should probably be indented.  Here and in the code below.